### DISCUSSION TITLE
Thank you to the team responsible for this.

### DISCUSSION THREAD

**Discussion by rkeniger** [dirkwhoffmann/vAmiga#740] [General]
I just want to say that vAmiga is everything I've ever wanted in a macOS Amiga emulator. The UI is fantastic, emulation works very well for the most part and most of all it's incredibly easy to get working.

Thank you, thank you, thank you. I hope you continue with the project because it's _awesome_.

**Comment by macbugs** [comment]
This is indeed an impressive work that exceeds expectations.  It is also rare to see quality [documentation](https://dirkwhoffmann.github.io/vAmiga/) like this in a small open-source project.  Like the original Amiga design, this emulator exhibits exceptional ability, and the project is a good example of how to utilize GitHub correctly.
